docs/source/array-design.rst

```diff
@@ -33,6 +36,14 @@ key-value pairs required to eventually compute the desired values, e.g.
     ...
    }
 
+The name of an ``Array`` object can be found in the ``name`` attribute.  One
+can get a nested list of keys with the ``._keys()`` method.  One can flatten
```

Comment:
._keys isn't user-level API but it's become a standard among
array/bag/dataframe.  I don't foresee changing it in the near future.
Thoughts?

On Fri, Jul 17, 2015 at 9:29 AM, Stephan Hoyer <notifications@github.com>
wrote:

> In docs/source/array-design.rst
> <https://github.com/ContinuumIO/dask/pull/436#discussion_r34905975>:
>
> > @@ -33,6 +36,14 @@ key-value pairs required to eventually compute the desired values, e.g.
> >      ...
> >     }
> >
> > +The name of an ``Array`` object can be found in the ``name`` attribute.  One
> > +can get a nested list of keys with the ``._keys()`` method.  One can flatten
>
> I've been wondering -- is this actually API? I've always been reluctant to
> use this method in external code because of the underscore prefix.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ContinuumIO/dask/pull/436/files#r34905975>.
>